apps/s_time.c

```diff
@@ -310,13 +312,12 @@ int s_time_main(int argc, char **argv)
     }
     totalTime += tm_Time_F(STOP); /* Add the time for this iteration */
 
-    i = (int)((long)time(NULL) - finishtime + maxtime);
```

Comment:
I couldn't find anywhere that this value was used.
It seems plausible that at one point it was used in the `printf` call a couple lines later, but that the expression got duplicated as part of a cleanup change; I did not look into the history to try to confirm that.